lib/utils/report.mjs

```diff
@@ -191,9 +191,9 @@ function applyFix({ fix, fixedNode, line, result: { stylelint }, ruleName }) {
 
 	if (!mayFix) return { applied: false };
 
-	fix();
-
 	if (!config.computeReplacementText) {
```

Comment:
once this is not a draft anymore
minor

Since we have `if (!isFn(fix))` at l185 and `if (!mayFix)` at l190, this `if` implies an `else` that throws an informative message for the user.
e.g. `"Both … have been requested resulting in a conflict; please turn off one to proceed."` (flanked by a link to the documentation)